testing/features/header.feature

```diff
@@ -9,6 +9,8 @@ Feature: Header component
     Background:
       Given a Standard Header component is on the page
 
+    # This specification is deliberately removed from Mobile Devices
```

Comment:
Worth nothing that "deliberately" is too strong. Links _should_ be accessible on small screens but aren't currently we have https://citizensadvice.atlassian.net/browse/NP-1235 to fix this so maybe tag this with that ID?

Reply:
So do we envisage this behaviour returning in 1235. Or different behaviour appearing? If it's the former I'll tag it as a future release/failing thing. If it's the latter I don't know what to do. maybe just mark it as a future release.

Reply:
Future release would be fine, it will be different behaviour